packages/react/src/components/Slider/Slider.stories.js

```diff
@@ -174,6 +175,33 @@ Default.args = {
   warnText: 'Warning message goes here',
 };
 
+export const Test = () => {
+  const [showSlider, setShowSlider] = useState(false);
+  const [sliderValue, setSliderValue] = useState(50);
+  return (
+    <>
+      <Checkbox
+        labelText="Show Slider"
+        id="showSlider"
+        checked={showSlider}
+        onChange={(checked) => setShowSlider(!showSlider)}
+      />
+      {showSlider
+        ? showSlider && (
```

Comment:
wouldn't `showSlider` on L190 be redundant since it's being evaluated on L189?
```suggestion
      {showSlider
        ? (
```

Reply:
This is a test story, I will remove it before merge.